ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/Image/ImagePage.tsx

```diff
@@ -93,14 +94,28 @@ function ImagePage() {
             </PageSection>
         );
     } else {
+        const digest = imageData?.metadata?.v1?.digest;
         mainContent = (
             <>
                 <PageSection variant="light">
                     {imageData ? (
-                        <Flex direction={{ default: 'column' }}>
-                            <Title headingLevel="h1" className="pf-u-mb-sm">
+                        <Flex
+                            direction={{ default: 'column' }}
+                            alignItems={{ default: 'alignItemsFlexStart' }}
+                        >
+                            <Title headingLevel="h1" className="pf-u-m-0">
                                 {imageName}
                             </Title>
+                            {digest && (
+                                <ClipboardCopy
+                                    hoverTip="Copy SHA"
+                                    clickTip="Copied!"
+                                    variant="inline-compact"
+                                    className="pf-u-display-inline-flex pf-u-align-items-center pf-u-mt-sm pf-u-mb-md pf-u-font-size-sm"
+                                >
```

Comment:
+1 for leveraging the PatternFly way of doing this.

> We will encourage you to develop the three great virtues of a programmer: laziness, impatience, and hubris.
Larry Wall, creator of Perl